iree/turbine/kernel/ops/wave_ops.py

```diff
@@ -751,9 +751,8 @@ def custom_string(self, value_map: dict[str, str]) -> str:
     def indexing_dims(self) -> list[IndexSymbol]:
         return list(self._type.symbolic_shape) if self._type else []
 
-    @property
-    def type(self) -> "Memory":
-        return self._type
+    def infer_type(self):
+        self.fx_node.type = self._type
```

Comment:
I do not see this change used in the rest of the PR, is this related?

Reply:
yes, I had to implement infer_types for these ops to get the scheduling to work.